src/sdk/workspaceloader.cpp

```diff
@@ -160,10 +167,23 @@ bool WorkspaceLoader::Open(const wxString& filename, wxString& Title)
         proj = proj->NextSiblingElement("Project");
     }
 
-    if (failedProjects > 0)
-    {
-        cbMessageBox(wxString::Format(_("%d projects could not be loaded.\nPlease see the Log window for details"), failedProjects),
-                     _("Opening WorkSpace"), wxICON_WARNING);
+    if (iFailedProjects > 0)
+    {
+        wxString sMessage = wxString::Format(_("%d projects could not be loaded.\nPlease see the Log window for details.\nThe failed projects are: %s"), iFailedProjects, sFailedProjects);
+        if (    (iFailedProjects == 1) &&
+                sFailedProjects.Contains("FortranProject") &&
+                (
+                    filename.Contains("CodeBlocks_wx31_64.workspace") ||
+                    filename.Contains("CodeBlocks_wx31.workspace")

```

Comment:
So the only purpose of codeblocks is to develop codeblocks on Windows. Please remove this.

Reply:
I put this in as I do not have the fortran source code, so every time I do a batch build I do not want the pop up to indicate the Fortran project could not be loaded.

I will remove it.

Reply:
Code has been pushed to resolve this.